### PR TITLE
Expanded on README

### PR DESCRIPTION
**Pull request by beccam**
- Added Contributor, Objectives, How this works sections. 
- Added link to Yarn pkg manager

**Question: Does this README need instructions on how to add Astra credentials locally? Unless I missed it, I do not see this step.**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,22 @@
...
+* Provide a fullstack development example using GraphQL with Astra as the storage backend.
+
+## How this works
+Once the Astra credentials are provided, the necessary tables are created in the database. The webservice will be available on port 8080 once the application has been deployed.
+
+This example uses Typescript. TypeScript is a typed superset of JavaScript that compiles to plain JavaScript, and adds optional static typing to the language. It is is designed for development of large applications and transcompiles to JavaScript.
+
+[Gatsby](https://www.gatsbyjs.org/) is a React-based, GraphQL powered, static site generator, with much of the functionality of a modern front-end framework. It brings together the best components of React, webpack, react-router, GraphQL, and other front-end tools. This is used to generate the application.
+
+Deploying a static web application on [Netlify](https://www.netlify.com/) allows us to reaps the benefits and not having to rely on our own webserver; it will also handle the Astra user credentials for us.
```
Comment (alexleventer) [inline README.md]: reaps -> reap

File: README.md [hunk 2]
```diff
@@ -19,7 +33,7 @@ A sample React, Typescript, DataStax Astra, Gatsby.js app starter app.
 ![image](https://user-images.githubusercontent.com/3254549/88746708-01e2c500-d102-11ea-9b25-174a45ef8545.png)
 5. Install Node dependencies: `yarn install`
 6. Run the set up script: ` . ./gitpod-setup.sh`
-7. Start your app by running: `yarn start`
+7. You will need [Yarn](https://yarnpkg.com/) to run this app. Start your app by running: `yarn start`
```
Comment (alexleventer) [inline README.md]: We could also add instructions to use npm (npm start)

File: README.md [hunk 3]
```diff
@@ -1,8 +1,22 @@
 # Astra/Gatsby.js Starter
-A sample React, Typescript, DataStax Astra, Gatsby.js app starter app.
+A sample React, Typescript, [DataStax Astra](https://astra.datastax.com/register), Gatsby.js app starter app.
+
+Contributor: [Sebastián Estévez](https://github.com/phact)
```
Comment (alexleventer) [inline README.md]: I actually wrote this one

File: README.md (hunk 1 above)
Comment (beccam) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (beccam) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (beccam) [reply README.md]: fixed

**Pull request merged**